src/Demo.tsx

```diff
@@ -14,7 +14,10 @@ export interface DemoState {
   infoText: string;
 }
 
-export default class Demo extends React.Component<void, DemoState> {
+export default class Demo extends React.Component<any, DemoState> {
+  state: DemoState;
+  setState: any;
```

Comment:
These are slightly annoying and I'm not sure how to best "fix" these yet. I think this is fine for now and could be looked at in a different PR.

Reply:
Did you update the React typings too?

Reply:
No I did not. That's a different issue I wanted to take care of later by introducing `@typings` and wanted to just isolate updating webpack.